plugins/github-pull-requests-board/src/utils/functions.ts

```diff
@@ -107,3 +121,28 @@ export const formatPRsByReviewDecision = (
     { title: COLUMNS.APPROVED, content: reviewDecisions.APPROVED },
   ];
 };
+
+export const shouldDisplayCard = (
+  repository: Repository,
+  author: Author,
+  teamRepositories: string[],
+  teamMembers: string[],
+  infoCardFormat: PRCardFormating[],
+  isDraft: boolean,
+) => {
+  // hide draft PRs unless "draft" filter is toggled
+  if (infoCardFormat.includes('draft') !== isDraft) {
+    return false;
+  }
+
+  const fullRepoName =
+    `${repository.owner.login}/${repository.name}`.toLocaleLowerCase('en-US');
```

Comment:
does this lowercasing work? does teamRepositories always contain already-lowercased things? i was a bit surprised at this one

Reply:
I double-checked a bit, and that's what I'm getting:
* `teamRepositories` comes from `useUserRepositoriesAndTeam` which gets a list of repositories from the Backstage API. It seems like the Backstage API returns everything lowercase
* `fullRepoName` is built using the data returned by the GraphQL data, which contains the original case from Github

For example, github will return `PagerDuty` as the owner, but Backstage will return `pagerduty`.

Would you like me to add a `const repositories = teamRepositories.map(repository => repository.toLocaleLowerCase('en-US'))` just in case?

Reply:
Aight, all good then